### PR TITLE
Feature/create account

### PR DESCRIPTION
**Pull request by Menyten**
Created created account view
Create account box
Inputs etc.

### CODE REVIEW

File: frontend/src/components/index.js [hunk 1]
```diff
@@ -1,11 +1,17 @@
 import React from 'react';
 import Start from '../views/Start';
+import Login from '../views/Login';
+import CreateAccount from '../views/CreateAccount';
 
 const HOME = () => <Start />;
+const LOGIN = () => <Login />;
+const CREATE_ACCOUNT = () => <CreateAccount />;
 
 const NotFound = () => <p>Denna routen finns inte!</p>
```
Comment (BrandtNick) [inline frontend/src/components/index.js]: English

File: frontend/src/views/CreateAccount/StyledCreateAccount.js [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+export const StyledGlobeIcon = styled(Globe)`
+  color: #fffaff;
+  width: 1.2rem;
+  margin-right: 0.5rem;
+  vertical-align: bottom;
+`;
+
+export const StyledClipboardIcon = styled(Clipboard)`
+  color: #fffaff;
```
Comment (BrandtNick) [inline frontend/src/views/CreateAccount/StyledCreateAccount.js]: This color exists many many times, we should add all colors to constants.

File: frontend/src/views/Login/Login.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+      <StyledLoginContainer>
+        <StyledLoginForm>
+          {renderInputs()}
+          <StyledLoginButton>
+            Logga in
+          </StyledLoginButton>
+        </StyledLoginForm>
+      </StyledLoginContainer>
+      <StyledTextContainer>
+        <StyledText>Har du inget konto? <StyledLink to="/create-account">Skapa ett nytt här.</StyledLink></StyledText>
```
Comment (BrandtNick) [inline frontend/src/views/Login/Login.js]: routes should be constants

File: frontend/src/views/CreateAccount/CreateAccount.js [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+  StyledOption
+} from './StyledCreateAccount';
+
+const CreateAccount = () => {
+
+  const renderOptions = () => countries.map(country => {
+    return <StyledOption value={country} key={country}>{country}</StyledOption>
+  });
+
+  const renderIcons = label => {
```
Comment (BrandtNick) [inline frontend/src/views/CreateAccount/CreateAccount.js]: nasty.. remove

File: frontend/src/views/Login/Login.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+      <StyledLoginContainer>
+        <StyledLoginForm>
+          {renderInputs()}
+          <StyledLoginButton>
+            Logga in
+          </StyledLoginButton>
+        </StyledLoginForm>
+      </StyledLoginContainer>
+      <StyledTextContainer>
+        <StyledText>Har du inget konto? <StyledLink to={CREATE_ACCOUNT}>Skapa ett nytt här.</StyledLink></StyledText>
```
Comment (BrandtNick) [inline frontend/src/views/Login/Login.js]: english

File: frontend/src/staticData.js [hunk 6]
```diff
@@ -1,3 +1,53 @@
...
+  , "Mauritius", "Mexico", "Moldova", "Monaco", "Mongolia", "Montenegro", "Montserrat", "Morocco", "Mozambique", "Namibia", "Nepal", "Netherlands", "Netherlands Antilles", "New Caledonia"
+  , "New Zealand", "Nicaragua", "Niger", "Nigeria", "Norway", "Oman", "Pakistan", "Palestine", "Panama", "Papua New Guinea", "Paraguay", "Peru", "Philippines", "Poland", "Portugal"
+  , "Puerto Rico", "Qatar", "Reunion", "Romania", "Russia", "Rwanda", "Saint Pierre &amp; Miquelon", "Samoa", "San Marino", "Satellite", "Saudi Arabia", "Senegal", "Serbia", "Seychelles"
+  , "Sierra Leone", "Singapore", "Slovakia", "Slovenia", "South Africa", "South Korea", "Spain", "Sri Lanka", "St Kitts &amp; Nevis", "St Lucia", "St Vincent", "St. Lucia", "Sudan"
+  , "Suriname", "Swaziland", "Sweden", "Switzerland", "Syria", "Taiwan", "Tajikistan", "Tanzania", "Thailand", "Timor L'Este", "Togo", "Tonga", "Trinidad &amp; Tobago", "Tunisia"
+  , "Turkey", "Turkmenistan", "Turks &amp; Caicos", "Uganda", "Ukraine", "United Arab Emirates", "United Kingdom", "Uruguay", "Uzbekistan", "Venezuela", "Vietnam", "Virgin Islands (US)"
+  , "Yemen", "Zambia", "Zimbabwe"];
+
 const Links = [
```
Comment (BrandtNick) [inline frontend/src/staticData.js]: capitilized.. why?

File: frontend/src/views/CreateAccount/CreateAccount.js [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+  StyledCreateButton,
+  StyledH3,
+  StyledSelect,
+  StyledOption,
+  StyledIcon
+} from './StyledCreateAccount';
+
+const CreateAccount = () => {
+
+  const renderOptions = () => countries.map(country => {
```
Comment (BrandtNick) [inline frontend/src/views/CreateAccount/CreateAccount.js]: implicit return

File: frontend/src/views/CreateAccount/CreateAccount.js [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+    return <StyledOption value={country} key={country}>{country}</StyledOption>
+  });
+
+  const renderInputs = () => createAccountInputs.map(({ id, label, type, icon }) => {
```
Comment (BrandtNick) [inline frontend/src/views/CreateAccount/CreateAccount.js]: implicit return

File: frontend/src/staticData.js (hunk 6 above)
Comment (Menyten) [reply frontend/src/staticData.js]: Dunno mah dude, wasn't me

File: frontend/src/views/CreateAccount/CreateAccount.js (hunk 8 above)
Comment (Menyten) [reply frontend/src/views/CreateAccount/CreateAccount.js]: menar du att jag ska lägga till return?

File: frontend/src/views/CreateAccount/CreateAccount.js (hunk 7 above)
Comment (Menyten) [reply frontend/src/views/CreateAccount/CreateAccount.js]: menar du att jag ska lägga till return?

**Pull request merged**